backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/DatabaseHandlerUtil.java

```diff
@@ -447,12 +453,33 @@ public static String convertObjectToJson(Object obj) {
           }
         };
 
+        Properties prop = CommonUtils.loadProperties(DatabaseSettings.class, PROPERTIES_FILE_PATH);
+        boolean ischangelogenabled = Boolean.parseBoolean(prop.getProperty("enable.sw360.change.log", "false"));
+        if (!ischangelogenabled) {
+	        if (!isChangeLogDisabledMessageLogged) {
+	            log.info("sw360change log is disabled");
```

Comment:
Removed the getProperty method, and I have added this isChangeLogDisabledMessageLogged variable to avoid the log message to print many times.

Reply:
1. Not sure. what is the need of the log to be printed once?
2. Info message is not correct, changelog is still active
3. return should happen at the beginning of the method not in the middle

Reply:
Thanks @JaideepPalit , for the comments, i have moved the return block to the beginning of this method, also i am printing the log once to not print the logs every time when we edit or update a project.

Regarding the 2nd comment, i am not sure if you are referring to the below logs in the console, if so this is part of the change log to update in the DB and it will not right any change log to the file if it is disable, this is called in **addChangeLogs** method.
2021-11-19 08:13:42 INFO DatabaseHandlerUtil:517 - Generating ChangeLogs.
2021-11-19 08:13:42 INFO DatabaseHandlerUtil:400 - Initialize ChangeLogs for Document Id : *********
2021-11-19 08:13:42 INFO DatabaseHandlerUtil:707 - Record Changes For Existing Doc.